testing/adios2/engine/SmallTestData.h

```diff
@@ -63,7 +63,7 @@ struct SmallTestData
          std::complex<double>(18.2, 19.2), std::complex<double>(19.2, 20.2)}};
 };
 
-SmallTestData generateNewSmallTestData(SmallTestData in, int step, int rank,
+SmallTestData generateNewSmallTestData(SmallTestData in, size_t step, int rank,
```

Comment:
I think this is triggering the VS failures. I typically just cast to `int` in the test.

Reply:
Yeah, I just didn't have time to look at it last night before I left.  I'll clean it up this morning though.